src/promptflow-azure/CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.9.0 (Upcoming)
+## 1.9.0 (2024.04.17)
```

Comment:
add a new 1.10 entry with Upcoming so devs know where to put new changelog

Reply:
Updated